.eslintrc.json

```diff
@@ -107,6 +106,7 @@
     "react/jsx-props-no-spreading": "off",
     "react/require-default-props": "off",
     "no-use-before-define": "off",
-    "@typescript-eslint/no-use-before-define": ["error"]
+    "@typescript-eslint/no-use-before-define": ["error"],
+    "@typescript-eslint/return-await": "off"
```

Comment:
The project has never used that rule and it only just started popping up as an error for some reason

Reply:
hmm, I think it'd be valuable to add in the end (some time in the future, in another ticket, far far away), to utilize better stack-tracing it seems on errors

https://v8.dev/blog/fast-async#improved-developer-experience

Reply:
Interesting, I never knew that. That being said, I don't think our stack traces are too crazy and it seems there's a performance difference using the directly returned version: https://stackoverflow.com/a/55080990

The standard from looking around is `return asyncfunc()` rather than `return await asyncfunct()` and the rest of the app uses the former version as well.